### PR TITLE
maintainers: remove superherointj

### PR DESCRIPTION
**Pull request by superherointj**
I'm sorry. But I no longer have the health/energy/time/sanity for this.

Thanks for the kindness.

Best regards.

## Pending

- [x] Revoke committer credential.
- [x] Review/merge back-port: #337585

### CODE REVIEW

**Review by emilazy — APPROVED** [review]
Thanks for your contributions to the project. Sorry to see you go.

**Comment by superherointj** [comment]
@emilazy Thanks for the merge.

File: pkgs/applications/networking/cluster/k3s/docs/ONBOARDING_MAINTAINER.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ If you cause a regression (we've all been there), you are responsible for fixing
 
 To merge code, you need to be a committer, or use the merge-bot, but currently the merge-bot only works for packages located at `pkgs/by-name/`, which means, K3s still need to be migrated there before you can use merge-bot for merging. As a non-committer, once you have approved a PR you need to forward the request to a committer. For deciding which committer, give preference initially to K3s committers, but any committer can commit. A committer usually has a green approval in PRs.
 
-K3s's committers currently are: superherointj, marcusramberg, Mic92.
+K3s's committers currently are: marcusramberg, Mic92.
```
Comment (Mic92) [inline pkgs/applications/networking/cluster/k3s/docs/ONBOARDING_MAINTAINER.md:29]: @marcusramberg You are still interested in maintaining k3s? I am still considering myself more of a part-time user.

Comment (marcusramberg) [reply pkgs/applications/networking/cluster/k3s/docs/ONBOARDING_MAINTAINER.md:29]: I'm kinda in the same boat, I just run a single node cluster on my desktop at the moment, accepted to join this to spread the load, so I'm not super eager to be the main driver ;-) 

Really sorry to see you go, @superherointj - thanks for all the contributions.

**Comment by emilazy** [comment]
@Mic92 Are you able to handle moving from committers to the retired contributors team, as requested in the PR message?

**Comment by Mic92** [comment]
I removed him from the committers team, but I cannot manage the retired contributors team yet. I will ask for access.

**Comment by Mic92** [comment]
Invite sent.

**Pull request merged**